localstack-core/localstack/services/events/target.py

```diff
@@ -390,6 +391,10 @@ class EventsTargetSender(TargetSender):
     def send_event(self, event):
         # TODO add validation and tests for eventbridge to eventbridge requires Detail, DetailType, and Source
         # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/events/client/put_events.html
+        target_arn = self.target["Arn"]
+        if ":api-destination/" in target_arn or ":destination/" in target_arn:
+            send_event_to_api_destination(target_arn, event, self.target.get("HttpParameters"))
```

Comment:
@maxhoheiser Might be something were instrumentation for EventStudio  might be a good idea in a follow up